tests/integration/components/modals/event-delete-modal-test.js

```diff
@@ -0,0 +1,13 @@
+import { test } from 'ember-qunit';
+import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('modals/event-delete-modal', 'Integration | Component | modals/event delete modal');
+
+test('it renders', function(assert) {
+  this.set('isOpen', false);
+  this.set('eventName', 'sample');
+  this.set('deleteEvent', 'deleteEvent');
```

Comment:
A string is not an action.
See:
https://guides.emberjs.com/v2.13.0/testing/testing-components/#toc_testing-actions